packages/simulators/lib/simulator_manager.dart

```diff
@@ -228,6 +228,28 @@ class IosSimulator {
     return;
   }
 
+  /// Take the screenshot of the simulator.
+  ///
+  /// Note that, this method takes screenshot of the entire simulator, not the
```

Comment:
nit: "note that" isn't necessary.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Reply:
done. thanks for sharing the link.